### ISSUE TITLE
Status pop-up window does not close under expected conditions.

### ISSUE THREAD

**Issue by kauffy** [meganz/MEGAsync#202]
Mint 18.3-Ubuntu, Cinnamon.
To reproduce:
1) Click the MegaSync icon in the system tray.
2) Select Status.
3) Status pops open, showing current, correct information.
4) Clicking outside of this window, on this window, into another application, or anything else, does not cause it to close. The pop-up remains in place, even if switching workspaces. The pop-up has not crashed, as all of its functionality is still intact.
5) In my tests on two machines, it closed one time by left-clicking on the MegaSync icon, but the rest of the times, clicking the MegaSync icon just opened the menu behind the Status pop-up.
-- On one machine, the edge of the menu was visible enough to select the top option (Status) which caused the status to close.
-- On the other machine, the menu was entirely behind the status pop-up, so couldn't be selected.

**Comment by polmr** [comment]
Hi @kauffy , What do you mean with `18.3-Ubuntu`? Ubuntu??
I've just tried an up-to-date Mint 18.3 with Cinnamon. MEGAsync works as expected. Left click on the systray icon opens the main UI and clicking outside it closes the pop-up. Please, ensure you have an up-to-date system and the latest megasync downloaded from https://mega.nz/sync.

**Comment by kauffy** [comment]
@polmr Yeah, a little confusing-- Mint 18.3 with the Ubuntu base (which, I now realize, is the only Mint 18.3, as contrasted with LMDE, which has a different version number).

I am definitely up-to-date on the system side, and I will check to make sure that the latest Megasync is installed, though I've only been using Megasync for about two weeks, and I had this problem from the start.

For certain, though, the behavior you describe is not what I experience-- it only _some_ times works that way. Much of the time, clicking the icon opens the pop-up, but clicking outside of it most definitely does not close it. Once in a while, clicking on the systray icon again will close it, but more often, clicking the systray icon just opens the systray menu.

Let me mess with a bit and triangulate.

**Comment by dsg38** [comment]
I'm on Ubuntu 18.04 and the Mega sync window pops up every time i open a notification from slack (i.e. click on the datetime at the top, then the notification). May also happen for other all notification types - not sure yet

**Comment by polmr** [comment]
We are trying to get to the bottom of this. We have been able to reproduce the issue with the never closing popup. It seems as if Qt/the desktop environment is not properly handling the event of clicking elsewhere than the open popup (status windows). It's odd, since it seems to be working fine when the popup is opened via trying to execute MEGAsync again instead of clicking in the systray and then Show status. Both ways in the end trigger the same code. We'll try to figure out a workaround for that. Regarding the issue with the notifications, it's been reported here too: https://github.com/meganz/MEGAsync/issues/173#issuecomment-396842132 . Some say it's related to qt 5.11 version, but Ubuntu 18.04 uses qt 5.9. I'm not sure if these two issues are related. We haven't been able to reproduce it. @dsg38,  does it occur to you with any notification? Can you try with this one for example (execute in a terminal): 
```
notify-send --hint=int:transient:1 -u low -t 100 "Notification"
```

**Comment by dsg38** [comment]
@polmr no it doesn't occur with all notifications. Currently noticed it with slack and 'Checker Plus for Gmail' chrome extension notification (possibly all chrome notifications).

Afraid the 'notify-send' notification doesn't trigger the same response.

Using the 'qtdiag' command, it looks like i have version 5.6.2:

~> qtdiag
Qt 5.6.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.2.0) on "xcb" 
OS: Ubuntu 18.04 LTS [linux version 4.15.0-22-generic]

**Comment by Lenbok** [comment]
I have the opposite problem, my window manager (windowmaker on 18.04) uses focus-follows-mouse, and as soon as the mouse crosses another window on the way from the systray icon where I triggered the popup window to the status window itself, it auto-closes. The pop up window shouldn't disappear until I click somewhere else.
